pkg/util/docker/dockerfile/dockerfile.go

```diff
@@ -0,0 +1,55 @@
+package dockerfile
```

Comment:
oh yeah... the `doc.go` file has been there all along... uncommited :yum: sorry.